assets/airports/airportLoadList.json

```diff
@@ -82,7 +82,7 @@
         "level": "hard",
         "name": "London Heathrow Airport",
         "premium": false,
-        "disabled": true
+        "disabled": false
```

Comment:
Unintentional I assume?

Reply:
yeah, just messing around and forgot to fix